extensions.json

```diff
@@ -11,7 +11,7 @@
     "version": "1.0.0",
     "sha256sum": "fa2b11af8ed7c420ca6e541196bca608661c0c1a81cd1f768c565c72a55a63c8",
     "foxe": "https://github.com/foxglove/blank-panel-extension/releases/download/1.0.0/foxglove.blank-panel-extension-1.0.0.foxe",
-    "keywords": ["blank","panel","empty","logo","spacer"]
+    "keywords": ["blank", "panel", "empty", "logo", "spacer"]
```

Comment:
Is the whitespace change intentional? I don't have a preference either way as long as this doesn't cause flapping.

Reply:
Changing back & forth. I'm not sure of the edit history here, but if the next editor saves & removes these spaces again, it can distract. Not a big deal.